src/main/java/com/denizenscript/clientizen/serverevents/ServerEvent.java

```diff
@@ -0,0 +1,18 @@
+package com.denizenscript.clientizen.serverevents;
+
+import com.denizenscript.clientizen.network.DataDeserializer;
+import com.denizenscript.clientizen.network.DataSerializer;
+
+public abstract class ServerEvent {
+
+	public boolean enabled;
+	public String id;
+
+	public void enable(DataDeserializer data) {}
+
+	public void disable() {}
```

Comment:
Are these two meant to be implemented in the subclasses? If so, mark them as `abstract` and remove their empty implementations

Reply:
They have an empty implementation to make it easier/cleaner for events which don't send/receive any data (e.g. a `clientizen player opens player inventory` event probably won't need any data)

Reply:
Added a comment clarifying that ^